tests/swift/TailorBearTest.py

```diff
@@ -24,22 +24,24 @@
 }
 """
 
-TailorBearWithoutConfig = verify_local_bear(TailorBear,
-                                            valid_files=(good_file,),
-                                            invalid_files=(bad_file,),
-                                            tempfile_kwargs={
+TailorBearWithoutConfigTest = verify_local_bear(TailorBear,
```

Comment:
move all of the parameters to the next line, indented four spaces, for all of these lines you have modified (but not other lines).

Reply:
Follow PEP 8 style guide please.